packages/terra-form-input/src/FormInput.scss

```diff
@@ -0,0 +1,103 @@
+:local {
```

Comment:
Left a comment on FormInput.jsx, lets rename this file to `Input.scss`

Reply:
Done 0fa82cf1ef4966c97404188b2b229c8f8385fb2b